scrabbler/simulation.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, player1, player2, start_player = None):
         # List of letters we can still  pick from.
         self.bag = list(Simulation.LETTERS)
         self.players = (player1, player2)
-        if start_player:
+        if start_player != None:
```

Comment:
This fix ensures that if you pass in `0` as the starting player, the starting player gets set properly.